upb/arena.h

```diff
@@ -80,6 +81,10 @@ UPB_INLINE size_t _upb_ArenaHas(upb_Arena* a) {
   return (size_t)(h->end - h->ptr);
 }
 
+UPB_INLINE uint_32_t upb_Arena_DebugRefCount(upb_Arena *arena) {
+  return arena->refcount;
```

Comment:
We need to find the group leader first. The correct algorithm here would be:

uint_32_t upb_Arena_DebugRefCount(upb_Arena *arena) {
  return arena_findroot(arena)->refcount;
}

Reply:
Indeed. I also forgot it in `upb_Arena_SpaceAllocated`. I added both.